mozci/util/taskcluster.py

```diff
@@ -6,11 +6,13 @@
 import functools
 
 import taskcluster_urls as liburls
+from taskcluster.helper import TaskclusterConfig
```

Comment:
You'll need to `poetry add` the taskcluster module.

Reply:
Oops, I guess I was lucky as `adr` depends on `taskcluster`